runtests.py

```diff
@@ -168,6 +168,50 @@ def store_variables(assignments, source, ns):
         else:
             variables.pop(variable, None)
 
+    
+
+def apply_xslt(row_number, row):
+    """
+    Process a CSV row as an XSLT method.
+    Payload is transformed by URI. Transformed files are stored in paths (described 
+        as /path/to/file=*) within Store. Rest of Store assignments are recorded.
+    row_number: CSV row number
+    row: Dictionary object that contains csv row data
+    """
+    xml_file, xslt_file = row['Payload'], row['URI']
+    
+    # separate outputs and assignments
+    output_files = []
+    assignments = []
+    for line in row['Store'].splitlines():
+        lhs, rhs = line.split('=', 1)
+        if rhs == '*':
+            output_files.append(lhs)
+        else:
+            assignments.append(line)
+    
+    assignments = '\n'.join(assignments)
+
+    with open(xml_file, 'rb') as source_file:
```

Comment:
This duplicates the functionality of a Method of "FILE".  Let's move the store operation to process the `output` after the transformation.